metering/metering-about-installing-metering.adoc

```diff
@@ -1,11 +1,11 @@
 [id="about-installing-metering"]
 = About installing Metering
 include::modules/common-attributes.adoc[]
-:context: about-installing-metering
+:context: metering-about-install
```

Comment:
I'd make the filename, ID, and context statement match.

Reply:
True - it is a requirement for id and context and a suggestion for filename: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata